Makefile

```diff
@@ -24,4 +24,10 @@ deps: trash
 
 .DEFAULT_GOAL := ci
 
-.PHONY: $(TARGETS)
\ No newline at end of file
+.PHONY: $(TARGETS)
+
+.PHONY:
+
+.PHONY: publish-linux-amd64
+publish-linux-amd64:
+	docker image push rancher/fluentd:${DRONE_TAG}-linux-amd64
```

Comment:
I don't even think we need the makefile :)

Reply:
fixed